app/components/common/TeamLeaderboardTable.component.js

```diff
@@ -1,52 +1,50 @@
-/* eslint-disable import/no-dynamic-require */
-/* eslint-disable global-require */
-import React from 'react';
-import PropTypes from 'prop-types';
-import { Table, List} from 'semantic-ui-react';
-import { FormattedNumber } from 'react-intl';
-import './style.scss';
+import React from "react";
+import PropTypes from "prop-types";
+import { Table } from "semantic-ui-react";
+import Avatar from "./Avatar.component";
+import "./style.scss";
 
-const TeamLeaderboardTable = ({
-  className, height, teams
-}) => (
-  <div>
-    <List style={{ height: height }} className={className}>
-      <Table celled collapsing basic='very'>
-        <Table.Header>
-          <Table.Row>
-            <Table.HeaderCell>#</Table.HeaderCell>
-            <Table.HeaderCell>Team Name</Table.HeaderCell>
-            <Table.HeaderCell>Distance</Table.HeaderCell>
-            <Table.HeaderCell>Finish Date</Table.HeaderCell>
+const TeamLeaderboardTable = ({ height, teams }) => (
+  <div style={{ height: height }} className={"leaderboard"}>
+    <Table collapsing basic="very" className="main">
+      <Table.Header>
+        <Table.Row>
+          <Table.HeaderCell>#</Table.HeaderCell>
+          <Table.HeaderCell></Table.HeaderCell>
+          <Table.HeaderCell>Team Name</Table.HeaderCell>
+          <Table.HeaderCell>Distance</Table.HeaderCell>
+          <Table.HeaderCell>Finish Date</Table.HeaderCell>
+        </Table.Row>
+      </Table.Header>
+      <Table.Body>
+        {teams.map((team, index) => (
+          <Table.Row key={team.name}>
+            <Table.Cell className="main position">{index + 1}.</Table.Cell>
+            <Table.Cell className="main">
+              <Avatar teamName={team.name} color={"#fa451b"} size={40} />
+            </Table.Cell>
+            <Table.Cell className="main team-name">{team.name}</Table.Cell>
+            <Table.Cell className="main distance">
+              {team.totalDistance}
+              km
+            </Table.Cell>
+            <Table.Cell
+              className={`main ${
+                team.completionDate ? "finish" : "pending"
+              } date`}
+            >
+              {team.completionDate ? team.completionDate : "Still to finish"}
+            </Table.Cell>
           </Table.Row>
-        </Table.Header>
-        <Table.Body>
-          {teams.map((item, index) => (
-            <Table.Row key={item.name} 
-              className={`${item.completionDate != undefined ? "finished-team-highlight" : ""}`}
```

Comment:
Is there a reason you removed this styling or based on completion date? Nick wanted the teams highlighted if they had finished hence added this styling. Maybe you have added it elsewhere. It doesn't appear to highlight if finish date exists?
![Screenshot 2021-04-28 at 14 12 31](https://user-images.githubusercontent.com/80352253/116410309-8cd92300-a82c-11eb-8cb0-e1ae3d52b307.png)

Reply:
It isn't highlighted in the design so I assumed Nick and Gavin had a chat about this. I'll confirm with them.